modules/ztp-sno-du-disabling-crio-wipe.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-sno-du-disabling-crio-wipe_{context}"]
+= Disable automatic CRI-O cache wipe
+
+After an uncontrolled host shutdown or cluster reboot, CRI-O automatically deletes the entire CRI-O cache,  causing all images to be pulled from the registry when the node reboots.
+This can result in unacceptably slow recovery times or recovery failures.
+To prevent this from happening in {sno} clusters that you install with {ztp}, disable the CRI-O delete cache feature during cluster installation.
```

Comment:
Should "sno" and "ztp" be capitalized?

Reply:
No, these are AsciiDoc attributes. See https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc#L154